src/launchers/steam.vala

```diff
@@ -169,7 +169,7 @@ namespace ProtonPlus.Launchers {
 
                     var root_array = root_node.get_array ();
 
-                    if (root_array.get_length () != 1)
+                    if (root_array.get_length () < 1)
```

Comment:
I used != since we used per_page=1 so if we get more than one result there`s a problem.

Reply:
Alright, I guess that makes sense